shared/common-adapters/tabs.tsx

```diff
@@ -88,30 +116,46 @@ const styles = Styles.styleSheetCreate(() => ({
       marginTop: 1,
     },
   }),
+  clickableBoxStyle: Styles.platformStyles({
+    isElectron: {
+      flexShrink: 1,
+      height: 40,
+      width: 120,
+    },
+    isMobile: {
+      height: 48,
+    },
+  }),
   container: {
+    backgroundColor: Styles.globalColors.white,
     borderBottomColor: Styles.globalColors.black_10,
```

Comment:
I think this border needs to be set on the scrollview as well, currently it ends at the edge of the screen. 
![image](https://user-images.githubusercontent.com/11968340/81231421-1f9c0480-8fc1-11ea-82c4-66b83150f589.png)